v2/server.go

```diff
@@ -94,48 +92,124 @@ type Server struct {
 	statusReceiveOverrideFn func(e cloudevents.Event, dataChan *channel.DataChan) error
 }
 
-// publisher/subscription data model
+// SubscriptionInfo
+//
+// SubscriptionInfo defines data types used for subscription.
+// swagger:parameters
+type SubscriptionInfo struct { //nolint:deadcode,unused
```

Comment:
These are defined in SDK right why was this brought  in here ?

Reply:
The structure defined has a different name `PubSub` in sdk-go, and it has `Version` to deal with v1/v2 encodings.
https://github.com/redhat-cne/sdk-go/blob/main/pkg/pubsub/pubsub.go#L35
It is not exactly same as the SubscriptionInfo defined in O-RAN Spec.
Maybe after 4.19 feature for removing v1 is done we can refactor this and remove `Version` from PubSub and rename it to SubscriptionInfo.